fern/definition/medical/document.yml

```diff
@@ -129,12 +136,42 @@ service:
             conversionType: "pdf"
           response:
             body: $DocumentUrl.Example1
+    
+    createDocumentReference:
```

Comment:
I followed the docx, but maybe we should decide to either call this createDocumentReference or uploadDocument and then stick with that for the entirety of the defintion instead of having it half and half

Reply:
This was a missing endpoint I added